Core/include/AnalysisMath.h

```diff
@@ -110,6 +110,110 @@ double Calculate_visiblePzeta(const LVector1& l1_p4, const LVector2& l2_p4)
 }
 
 
+template<typename Iterator>
+double Calculate_HT(Iterator begin, const Iterator& end){
+    auto sum = 0;
```

Comment:
"auto sum = 0" will be resolved as int. It is better to directly write
double sum = 0.;